book/art/fullnode.bob

```diff
@@ -1,27 +1,30 @@
-             .---------------------------.
-             |  Fullnode                 |
-             |                           |
- .--------.  |  .------------------.     |
- |        |---->|                  |     |
- | Client |  |  | JSON RPC Service |     |
- |        |<----|                  |     |
- `----+---`  |  `------------------`     |
-      |      |     ^                     |    .------------------.
-      |      |     |  .----------------. |    | Validators       |
-      |      |     |  | Gossip Service +----->|                  |
-      |      |     |  `--------+-------` |    |  .------------.  |
-      |      |     |         ^ |         |    |  |            |  |
-      |      |     |         | v         |    |  | Upstream   |  |
-      |      |  .--+---.   .-+---.       |    |  | Validators |  |
-      |      |  | Bank |<--| TVU |<--------------+            |  |
-      |      |  `------`   `-----`       |    |  `------------`  |
-      |      |     ^                     |    |                  |
-      |      |     |                     |    |  .------------.  |
-      |      |  .--+--.   .-----------.  |    |  |            |  |
-      `-------->| TPU +-->| Broadcast +--------->| Downstream |  |
-             |  `-----`   | Service   |  |    |  | Validators |  |
-             |            `-----------`  |    |  |            |  |
-             |                           |    |  `------------`  |
-             `---------------------------`    |                  |
-                                              `------------------`
-
+             .---------------------------------.
+             |  Fullnode                       |
+             |                                 |
+ .--------.  |  .------------------.           |
+ |        |---->|                  |           |
+ | Client |  |  | JSON RPC Service |           |
+ |        |<----|                  |           |
+ `----+---`  |  `------------------`           |
+      |      |     ^    .----------------.     |
+      |      |     |    | Gossip Service |     |  .------------------.
+      |      |     |    | (syncs votes & |<------>| Validators       |
```

Comment:
No need for `syncs votes & contact info`. We can explain that in the Gossip Service chapter.

Reply:
👍